### ISSUE TITLE
<Solved> from mediapipe.python._framework_bindings import 

### ISSUE THREAD

**Issue by chenlize96** [google/mediapipe#3122]
**Versions:**
python version - **3.8** (I did not use anaconda)
OS - **Windows** system terminal
the rest are all based on installation **tutorial**
Suppose you are not using 'pip3 install mediapipe' to install mediapipe. Instead, you use python package, and the last step is '**python setup.py install --link-opencv**'.

**Errors:**
from mediapipe.python._framework_bindings import resource_util ImportError: DLL load failed: The specified module could not be found.

**Solutions:**
If #1839 #1405 doesn't work for you, please check if you run 'import mediapipe' in the mediapipe repository.
(I do not need Visual C++ redistributable packages OR msvc-runtime)
**Reminder:** DO NOT run test ‘import mediapipe’ in the mediapipe repository
Firstly, you should run '**pip3 install absl-py attrs matplotlib opencv-contrib-python protobuf --user**' in your terminal since requirements.txt lacks some libraries.
Then, for me, I run 'import mediapipe' in **IDLE (Python 3.8 64-bit)**.
It should work!

**Last check step** ->
>> pip3 list
```
Package               Version
--------------------- ---------
matplotlib            3.5.1
mediapipe             dev
numpy                 1.22.2
opencv-contrib-python 4.5.5.62
opencv-python         4.5.5.62
...
```

There should not be version number here. Enjoy editing and creating your own mediapipe :)

I made this post since there are still many people asking the same question. I am glad to share my solution which is unqiue but effective.

**Comment by Akash17-coder** [comment]
I tried it in pycharm but it is showing version number

**Comment by chenlize96** [comment]
@Akash17-coder Have you inputted "pip3 uninstall mediapipe" before you installed the python package for mediapipe.

**Comment by sureshdagooglecom** [comment]
Hi @chenlize96 , Have a look at this https://github.com/google/mediapipe/issues/1839#issuecomment-817253475.

**Comment by sureshdagooglecom** [comment]
Hi @chenlize96 , can you provide steps to reproduce this issue.

**Comment by chenlize96** [comment]
@sureshdagooglecom 
run 'python setup.py install --link-opencv' in the Windows terminal.
it shows: 'from mediapipe.python._framework_bindings import resource_util ImportError: DLL load failed: The specified module could not be found.'
1. run 'pip install msvc-runtime', then 'python setup.py install --link-opencv'. I got the same error. Thus, I run 'pip uninstall msvc-runtime'
2. install Visual C++ redistributable packages, then 'python setup.py install --link-opencv'. I got the same error. Thus I uninstalled this package.
3. DO NOT run test ‘import mediapipe’ in the mediapipe repository
